models/model.js

```diff
@@ -49,6 +49,35 @@ selectArticlesById = (article_id) => {
         return result.rows;
       });
   };
+
+  insertComment = ({ username, body }) => {
+    console.log("in model")
+    return db
+        .query(`INSERT INTO comments(username, body) 
+            VALUES ($1, $2) RETURNING *`,[username, body])
+        .then((result) => {
+            console.log(result.rows)
+            return result.rows[0]
+        }).catch((err)=> {
+            console.log(err)
+        })
+  }
+
+  removeCommentById = (comment_id) => {
+    return db.query('DELETE FROM comments WHERE comment_id = $1;', [comment_id])
+    .then((result)=> {
+       if(result.rowCount === 0) return Promise.reject({msg: "Not found"})
+        return result
+    })
+  };
```

Comment:
Nice, but no need to even really send result back here if we dont do anything with it